packages/odata-common/src/filter/filterable.ts

```diff
@@ -75,8 +75,9 @@ export function and<
 ): FilterList<EntityT, DeSerializersT> {
   return new FilterList(transformVariadicArgumentToArray(first, rest));
 }
+/* eslint-enable jsdoc/require-returns-check, jsdoc/require-param, jsdoc/check-param-names */
 
-// eslint-disable-next-line jsdoc/require-returns-check
+/* eslint-disable jsdoc/require-returns-check, jsdoc/require-param, jsdoc/check-param-names */
```

Comment:
[q] Why not put this at the top

Reply:
I wanted to make sure that these rules are only disabled for these two functions in case we in the future write something new in this file.